Description: This relaxes the mysql2 version for tests.
Author: Utkarsh Gupta <guptautkarsh2102@gmail.com>
---

--- ruby-activerecord-import-0.28.1.orig/Gemfile
+++ ruby-activerecord-import-0.28.1/Gemfile
@@ -4,8 +4,8 @@ gemspec
 
 version = ENV['AR_VERSION'].to_f
 
-mysql2_version = '0.3.0'
-mysql2_version = '0.4.0' if version >= 4.2
+#mysql2_version = '0.3.0'
+#mysql2_version = '0.4.0' if version >= 4.2
 
 group :development, :test do
   gem 'rubocop'
@@ -14,7 +14,7 @@ end
 
 # Database Adapters
 platforms :ruby do
-  gem "mysql2",                 "~> #{mysql2_version}"
+  gem "mysql2"
   gem "pg",                     "~> 0.9"
   gem "sqlite3",                "~> 1.3.10"
   gem "seamless_database_pool", "~> 1.0.20"
